render-to-string.js

```diff
@@ -1,24 +1,16 @@
 const mdx = require('@mdx-js/mdx')
-const { MDXProvider } = require('@mdx-js/react')
+const { MDXProvider, mdx: mdxReact } = require('@mdx-js/react')
 const { transformAsync } = require('@babel/core')
 const presetEnv = require('@babel/preset-env')
 const presetReact = require('@babel/preset-react')
 const pluginBrowser = require('./babel-plugin-mdx-browser')
-const nodeEval = require('require-from-string')
 const reactRenderToString = require('react-dom/server').renderToString
 const React = require('react')
 
-module.exports = function renderToString(source, components, options) {
+module.exports = function renderToString(source, components, options, scope) {
   let jsSource
   // transform it into react
-  const renderer = `
-import React from 'react'
-import { mdx } from '@mdx-js/react'
-`
-  return mdx(source, options)
-    .then((jsx) => {
-      return `${renderer}\n${jsx}`
-    })
+  return mdx(source, { ...options, skipExport: true })
```

Comment:
I hadn't dug up this `skipExport` option, this is a great find. I think we can also remove [this transform](https://github.com/hashicorp/next-mdx-remote/blob/master/babel-plugin-mdx-browser.js#L28) safely now that it has been added!

Reply:
👍